certified-connectors/N-able Cloud User Hub/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5056 @@
+                          "commandSet": true,
+                          "commandBlock": true
+                        }
+                      }
+                    ]
+                  },
+                  "errorCode": "string",
+                  "isSuccess": true
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "There was missing or invalid input. The response body contains the error details.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "error": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "type": "string",
+                      "description": "Code.",
+                      "title": "Code"
+                    },
+                    "message": {
+                      "type": "string",
+                      "description": "Message.",
+                      "title": "Message"
+                    },
+                    "innerError": {
+                      "type": "object",
+                      "properties": {
+                        "request-id": {
+                          "type": "string",
+                          "description": "Request Id.",
+                          "title": "Request id"
+                        },
+                        "date": {
+                          "type": "string",
+                          "description": "Date.",
+                          "title": "Date"
+                        }
+                      },
+                      "description": "Inner error."
+                    }
+                  },
+                  "description": "Error."
+                }
+              }
+            },
+            "examples": {
+              "application/json": {
+                "error": {
+                  "code": "string",
+                  "message": "string",
+                  "innerError": {
+                    "request-id": "string",
+                    "date": "string"
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "The request was not authenticated. The client needs to authenticate with Cloud User Hub APIs first."
+          },
+          "403": {
+            "description": "The request was not authorized. The client needs to have authorization to access this resource."
+          },
+          "404": {
+            "description": "The specified resource was not found."
+          },
+          "500": {
+            "description": "Unexpected error."
+          }
+        }
+      }
+    },
+    "/command/v1/organizations/{organizationId}/commandblocks/{commandBlockId}/jobs/ForUser(targetUserId={targetUserId})": {
+      "post": {
+        "description": "Add a Command Block job, provider and action validation depend on the specific command block.",
```

Comment:
@VincentMejan please make sure that everything is grammatically correct.

Reply:
Fixed in latest commit